src/starknetCli.ts

```diff
@@ -344,15 +347,16 @@ export function runStarknetNewAccount(options: StarkNetNewAccountOptions) {
     : '';
   const accountDirOption = options.account_dir ? `--account_dir ${options.account_dir}` : '';
   try {
-    execSync(
+    const output = execSync(
       `${warpVenvPrefix} starknet new_account ${networkOption} ${walletOption} ${accountOption} ${gatewayUrlOption} ${feederGatewayUrlOption} ${accountDirOption}`,
       {
-        stdio: 'inherit',
         encoding: 'utf8',
       },
     );
-  } catch {
-    logError('Starknet new account creation failed');
+    console.log(output);
+  } catch (e: any) {
+    const error = getPersonalizedStarknetError('starknet new_account', e.stderr.toString());
+    logError(error);
   }
```

Comment:
We might want to build a wrapper around execSync, the pattern is repeating

Reply:
I mean, setting encoding, logging, adding installation hints

Reply:
there are places with no catch block like this
https://github.com/NethermindEth/warp/blob/94e1912f09760147ba2a986355a43d69b54df646/src/solCompile.ts#L148
So we should be returning the `stdout` and `stderr` to apply some logic there. Also the options change, take this as an example
https://github.com/NethermindEth/warp/blob/94e1912f09760147ba2a986355a43d69b54df646/src/autoRunSemanticTests.ts#L190
so we should then specify which options and their values to pass.

Reply:
We don't have to specialize the wrapper. We can simply use them where it's handy and stick with execSync whenever we need more customization :)